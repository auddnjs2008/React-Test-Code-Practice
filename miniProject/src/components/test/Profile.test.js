import { render } from "@testing-library/react";
import React from "react";
import Profile from "../Profile";

describe("<Profile/>", () => {
  it("mathces snapshot", () => {
    const utils = render(<Profile username="const" name="auddnjs" />);
    expect(utils.container).toMatchSnapshot();
  });

  it("shows the props correctly", () => {
    const utils = render(<Profile username="const" name="auddnjs" />);
    utils.getByText("const");
    utils.getByText("(auddnjs)");
    utils.getByText(/auddnjs/);
  });
});
