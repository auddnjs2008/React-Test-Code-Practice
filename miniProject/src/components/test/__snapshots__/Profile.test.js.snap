// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> mathces snapshot 1`] = `
<div>
  <div>
    <b>
      const
    </b>
    Â 
    <span>
      (
      auddnjs
      )
    </span>
  </div>
</div>
`;
